evm/logic/system.py

```diff
@@ -190,3 +191,69 @@ def __call__(self, computation):
         if computation.msg.is_static:
             raise WriteProtection("Cannot modify state while inside of a STATICCALL context")
         return super(CreateEIP150, self).__call__(computation)
+
+
+class Create2(CreateEIP150):
```

Comment:
Is there's a reason for inheriting `CreateEIP150` instead of `CreateByzantium`?

Reply:
`CreateByzantium` and `Create2` both override `__call__` function so I make `Create2` inherit from `CreateEIP150` and copy the `is_static` check from `CreateByzantium`.